content/webapp/utils/colors.ts

```diff
@@ -0,0 +1,16 @@
+import { ColorSelection } from '../types/color-selections';
+
+// Purple, Treal, Red and Green used to be offered in Prismic as options
+// We should still support legacy stories so this function helps transform the colors
+// Into the new ones
+export const getSeriesColor = (
```

Comment:
I think this should return a default colour if prismicColor is undefined? I'd be inclined to use a switch statement

Reply:
you're right, this was in the effort to have new colours but we don't have them anymore

Reply:
hmm, seems we set a default when we use it, but might be nicer to do it here and use it in the series transformer

Reply:
then we could just call getSeriesColor here: https://github.com/wellcomecollection/wellcomecollection.org/blob/e98e45561f0ae9459e4f9b33943dc382b3201413/content/webapp/services/prismic/transformers/series.ts#L15

Reply:
> then we could just call getSeriesColor here:
> 
> https://github.com/wellcomecollection/wellcomecollection.org/blob/e98e45561f0ae9459e4f9b33943dc382b3201413/content/webapp/services/prismic/transformers/series.ts#L15

I like the idea, but `color` is expected to be of type `ColorSelection`, which we've revert back to using the old ones... I could change that, but the idea is that we'll adapt ColorSelection later on, right?

Reply:
I think we could change ColorSelection to the new colours, use the getSeries function in the transformer so we return the correct thing. Then look at changing the prismic model, so the editors choose from the new colours, as a next step

Reply:
Got you :) Thanks for taking the time and pushing for better; this area is a bit new territory and I asked you last minute, I appreciate it ⭐ 
I pushed some changes, tests are passing, does it matches what you were thinking?